src/couch_mrview.erl

```diff
@@ -653,6 +653,18 @@ make_meta(Args, UpdateSeq, Base) ->
     end.
 
 
+get_total_rows(#db{local_tree = LocalTree} = Db, #mrargs{extra = Extra}) ->
+    case couch_util:get_value(namespace, Extra) of
+        <<"_local">> ->
+            FoldFun = fun(_, _, Acc) -> {ok, Acc + 1} end,
+            {ok, _, Total} = couch_btree:foldl(LocalTree, FoldFun, 0),
```

Comment:
Nope, `couch_btree:size/1` returns active size of btree in bytes and we need number of documents. For id btree we are taking advantage of the fact that we are keeping this info on full reduction in tree's root, but unfortunately we don't do that for local btree, so I have to fold here.